### PR TITLE
Obliterate Reusability

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: common/src/main/scala/explore/components/TileController.scala [hunk 1]
```diff
@@ -105,7 +103,7 @@ object TileController {
       }
       // Store the current layout
       .useStateViewWithReuseBy((p, _, _) => updateResizableState(p.layoutMap))
-      .renderWithReuse { (p, debouncer, bn, currentLayout) =>
+      .render { (p, debouncer, bn, currentLayout) =>
```
Comment (cquiroz) [inline common/src/main/scala/explore/components/TileController.scala:106]: In this case for example, it seems the Reuse does not complicate the code a lot

File: explore/src/main/scala/explore/ExploreLayout.scala [hunk 2]
```diff
@@ -27,89 +26,83 @@ import scala.scalajs.js
...
-  val view: ReuseView[RootModel]
+  val view: View[RootModel]
 ) extends ReactProps[ExploreLayout](ExploreLayout.component)
 
 object ExploreLayout {
   type Props = ExploreLayout
 
-  implicit val resolutionWithPropsReuse
-    : Reusability[ResolutionWithProps[Page, ReuseView[RootModel]]] =
+  implicit val resolutionWithPropsReuse: Reusability[ResolutionWithProps[Page, View[RootModel]]] =
```
Comment (toddburnside) [inline explore/src/main/scala/explore/ExploreLayout.scala:37]: Do we still need this?

File: common/src/main/scala/explore/components/TileController.scala (hunk 1 above)
Comment (rpiaggio) [reply common/src/main/scala/explore/components/TileController.scala:106]: I started by removing it from everywhere, and have a working version of Explore with that.

We can add it back in the places where it's needed, but I think we should have a uniform criterion.

For example, I notice the performance is not the same as before when resizing. We should probably add it back in the situations where it does have an impact.

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**